src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -86,10 +86,10 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
   const goToCommunityScreen = (channel: ChannelList) => {
     setChannelAsRead(channel);
-
     const topicName = channel?.name?.replace(/^#/, '');
     const navigationParam = {
-      id: convertTopicNameToTopicPageScreenParam(topicName)
+      id: convertTopicNameToTopicPageScreenParam(topicName),
+      channelPicture: channel.channelPicture
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here is adding the `channelPicture` to the `navigationParam`. This seems fine as long as the `TopicPageScreen` can handle this new parameter and use it appropriately. However, there's no null check for `channel.channelPicture`. If it's possible that `channel.channelPicture` could be `null` or `undefined`, you should handle this case to prevent potential errors.
```diff
-      channelPicture: channel.channelPicture
+      channelPicture: channel?.channelPicture ?? defaultChannelPicture
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->